smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/SmackIntegrationTestFramework.java

```diff
@@ -665,6 +676,10 @@ private static Exception throwFatalException(Throwable e) throws Error, NoRespon
         return (Exception) e;
     }
 
+    public interface TestRunResultProcessor {
```

Comment:
We probably could (and should) annotate this with `@FunctionalInterface`. Not sure if we already do this in Smack, but it is certainly not wrong doing so.